clients/admin-ui/cypress.config.ts

```diff
@@ -4,6 +4,15 @@ import { defineConfig } from "cypress";
 export default defineConfig({
   e2e: {
     baseUrl: "http://localhost:3000",
+    setupNodeEvents(on) {
+      on("before:browser:launch", (browser, launchOptions) => {
+        if (browser.family === "chromium") {
+          // No need for tests to be slowed down by animations!
```

Comment:
⭐ - also does it matter what browser the cypress test uses? Can we just set this across all browsers for e2e testing?

Reply:
I think it's more that the `--force-prefers-reduced-motion` option might be chromium specific. I believe we only use electron (which I think is chromium based) and chrome though, so that should cover our use cases!